primitive/interceptor.go

```diff
@@ -29,6 +29,10 @@ type Invoker func(ctx context.Context, req, reply interface{}) error
 // use type assert to get real type.
 type Interceptor func(ctx context.Context, req, reply interface{}, next Invoker) error
 
+var NoopInterceptor = func(ctx context.Context, req, reply interface{}) error {
```

Comment:
Why add this method?

Reply:
In my use, I want to trigger the interceptor as AfterPullMessageHook. [link code](https://github.com/apache/rocketmq-client-go/pull/745/commits/6e2b133181a6763d6f0aef746187d88aab9ae7e4#diff-3e4be2af9a7961a81e0a80bf3e76646ef3f82fc73ed04f636960e990f4d9260fR116)
But I don't need to do some operations after the interceptor ends.